TEST/regression-wiki.mjs

```diff
@@ -28,18 +29,17 @@ const UNSAFE_WIKI_FILES = new Set([
   "user-hdeyoung-monweakfoc-elf",
 ]);
 
+exec("node elf-watcher.mjs", { cwd: "../wiki/" });
 const WIKI_TWELF_LOC = "../wiki/pages/";
 const cfgs = [];
 for (const file of readdirSync(WIKI_TWELF_LOC)) {
-  if (file.endsWith(".elf")) {
+  if (file.endsWith(".cfg")) {
     const base = file.slice(0, file.length - 4);
     if (IGNORED_WIKI_FILES.has(base)) continue;
-    const cfg = base + ".cfg";
-    writeFileSync(WIKI_TWELF_LOC + cfg, file);
     cfgs.push(
       `test${
         UNSAFE_WIKI_FILES.has(base) ? "Unsafe" : ""
-      } ${WIKI_TWELF_LOC}${cfg}`
+      } ${WIKI_TWELF_LOC}${file}`
```

Comment:
Moving the cfg auto-generation into the wiki project means we can just invoke that auto-generation to run the regression tests.